docs/open-telemetry.asciidoc

```diff
@@ -51,6 +51,8 @@ APM Server.
 export OTEL_RESOURCE_ATTRIBUTES=service.name=checkoutService,service.version=1.1,deployment.environment=production
 export OTEL_EXPORTER_OTLP_ENDPOINT=https://apm_server_url:8200
 export OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer an_apm_secret_token"
+export OTEL_METRICS_EXPORTER="otlp" \
```

Comment:
According to https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#exporters, OTEL_METRICS_EXPORTER defaults to otlp. Do we need this?

Reply:
`OTEL_METRICS_EXPORTER` changes its default from `none` to `otlp` in the Otel Java SDK 1.13.0 which has not yet been released as an java agent. Hopefully in 2 or 3 weeks.
Moreover, I don't know when other otel SDKs will enable metrics by default.

For these reasons, I would like to document `OTEL_METRICS_EXPORTER=otlp` for some time.

Reply:
Makes sense, thanks for pointing that out.